spec/features/user_signup_functionality_spec.rb

```diff
@@ -82,7 +82,7 @@
       click_link("Logout")
       visit "/"
       click_link("Create Account")
-      fill_in('new_user[email]', with: 'Jeff@gmail.com')
+      fill_in('new_user[email]', with: 'Jason@gmail.com')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.